### PR TITLE
Disable PyTorch/XLA diffusers tests.

### PR DESCRIPTION
**Pull request by yeounoh**
# Description

Disable PyTorch/XLA diffusers tests.

# Tests

N/A

### CODE REVIEW

File: dags/pytorch_xla/nightly.py [hunk 1]
```diff
@@ -160,12 +160,13 @@ def huggingface():
...
-      ),
-      US_CENTRAL2_B,
-  ).run()
+  # TODO(yeounoh) the dataset is taken down, we need a new SD model with a new dataset.
+  # diffusers_v4_8 = task.TpuQueuedResourceTask(
+  #     test_config.JSonnetTpuVmTest.from_pytorch(
+  #         "pt-nightly-hf-diffusers-func-v4-8-1vm"
+  #     ),
+  #     US_CENTRAL2_B,
+  # ).run()
```
Comment (vanbasten23) [inline dags/pytorch_xla/nightly.py:169]: also delete the reference at line 172? Similarly for r2_3.py

**Review by JackCaoG — APPROVED** [review]

File: dags/pytorch_xla/nightly.py (hunk 1 above)
Comment (yeounoh) [reply dags/pytorch_xla/nightly.py:169]: thanks!

**Review by vanbasten23 — APPROVED** [review]

**Comment by vanbasten23** [comment]
Need to fix the format.

**Review by zpcore — APPROVED** [review]
Thanks for looking into the HF model issue.

**Pull request merged**